dart/dynamics/ShapeFrame.hpp

```diff
@@ -161,6 +161,9 @@ class DynamicsAspect final : public common::AspectWithVersionedProperties<
   DART_COMMON_SET_GET_ASPECT_PROPERTY( double, SlipCompliance )
   // void setSlipCompliance(const double& value);
   // const double& getSlipCompliance() const;
+  // Support API that matches upstream 6.10.0 release as well
```

Comment:
The upstream has the property named `mPrimarySlipCompliance`. Should we do the same for consistency and provide the additional functions `setSlipCompliance` and `getSlipCompliance`? I think it makes sense if we're going to deprecate these at some point.

Reply:
I took a minimal approach here that doesn't break API/ABI, while I believe changing the name of a public member variable would break both.

I actually don't think it's a big problem to break ABI because we are linking to DART with plugins, and I think most people are using the get/set methods rather than directly reading and writing to the public member variable. So I would be fine with it; I just took a minimal approach for this PR.

What if we merge this to add the missing APIs and then open a separate pull request to rename the member variable?

Reply:
huh, didn't realize those were public members. Making a minimal change here makes sense.